### ISSUE TITLE
Signing SAID's (`d` attr of any event) vs. signing the whole event

### ISSUE THREAD

**Issue by blelump** [WebOfTrust/keripy#371]
Hey,

since the `cesr-proof` and SAD path language is in place, would it make sense to sign just `d` attribute of any event rather than the whole event?

Then is CESR the serialized event along with signature would be: 
```
{ event }
-JAB
5AABAA-d
...
```

**Comment by SmithSamuelM** [comment]
Yes that makes sense. I think there is some code that would break in keripy to get there but there is a performance boost
from signing only the said.
